src/utils.jl

```diff
@@ -21,6 +21,22 @@ function unique_indices(x)
     return unique, indices
 end
 
+"""
+    split_chains(data::AbstractArray{<:Any,3}, split::Int=2)
```

Comment:
- Is it (supposed to be) exported? If not, maybe use `_split_chains` to make it clearer that it is an internal function?
- Do we need a default argument at all?

Reply:
> * Is it (supposed to be) exported?

Originally I planned for it to be, but I no longer thing that is useful.

> * If not, maybe use `_split_chains` to make it clearer that it is an internal function?
> 
>     * Do we need a default argument at all?

`split_chain_indices` similarly has no leading underscore and has a default argument, and because the methods are related, I think it makes sense to keep them consistent in this regard.

Reply:
While BDA3 and Vehtari's paper do not discuss the case with odd chains, Stan, posterior, and ArviZ all discard the middle element. For `split_chains > 2`, we could generalize this by discarding the first `r` middle elements, where `r = rem(niter, split_chains)`. This requires bringing back `copy_split!`.